docs/examples/content-type/README.md

```diff
@@ -112,6 +112,40 @@ response = requests.post(
 )
 ```
 
+### Codecs
+
+In order to prepare our V2 request payload, it's often better (and recommended) to use codecs.
```

Comment:
Can we rephrase as its hard to understand - if its recommended then should not be in brackets and the second sentence is hard to follow

Reply:
Good point!

I've rephrased that section, as well as added more info to the main content-types doc page. Would be great to hear your thoughts on the updated versions!